src/main/java/ch/njol/skript/effects/EffSendBlockChange.java

```diff
@@ -1,100 +1,95 @@
 package ch.njol.skript.effects;
 
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.block.Block;
-import org.bukkit.block.data.BlockData;
-import org.bukkit.entity.Player;
-import org.bukkit.event.Event;
-import org.jetbrains.annotations.Nullable;
-
 import ch.njol.skript.Skript;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.doc.Description;
-import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Example;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
 import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Send Block Change")
-@Description("Makes a player see a block as something it really isn't. BlockData support is only for MC 1.13+")
-@Examples({"make player see block at player as dirt",
-		"make player see target block as campfire[facing=south]"})
-@Since("2.2-dev37c, 2.5.1 (block data support)")
+@Description("Makes a player see a block as something else or as the original.")
+@Example("make player see block at player as dirt")
+@Example("make player see player's target block as campfire[facing=south]")
+@Example("""
+	make all players see (blocks in radius 5 of location(0, 0, 0)) as bedrock
+	make all players see (blocks in radius 5 of location(0, 0, 0)) as original
+	""")
+@Since("2.2-dev37c, 2.5.1 (block data support), INSERT VERSION (as original)")
 public class EffSendBlockChange extends Effect {
 
-	private static final boolean BLOCK_DATA_SUPPORT = Skript.classExists("org.bukkit.block.data.BlockData");
-	private static final boolean SUPPORTED =
-			Skript.methodExists(
-					Player.class,
-					"sendBlockChange",
-					Location.class,
-					Material.class,
-					byte.class
-			);
-
 	static {
 		Skript.registerEffect(EffSendBlockChange.class,
-				BLOCK_DATA_SUPPORT ? "make %players% see %blocks% as %itemtype/blockdata%" : "make %players% see %blocks% as %itemtype%"
+			"make %players% see %blocks% as %itemtype/blockdata%",
+			"make %players% see %blocks% as (original|normal)"
```

Comment:
```suggestion
			"make %players% see %blocks% as [the|its] (original|normal|actual) [block]"
```
some extra options
Also, this is optional, but would you be willing to swap %blocks% for %locations%? It shouldn't be breaking and should allow the expression to be a lot more performant at scale, since you don't ever need to get the block itself anymore.